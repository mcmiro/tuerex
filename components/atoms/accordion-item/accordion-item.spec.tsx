import * as React from "react";
import {screen, render} from "@testing-library/react";
import AccordionItem from ".";

describe("Test AccordionItem component", () => {
	it("Should pass when AccordionItem is defined", () => {
		render(
			React.cloneElement(
				<AccordionItem name="test" content="test" isActive={true} data-testid="accordion-test" />
			)
		);

		const accordionItem = screen.queryAllByTestId("accordion-test");
		expect(accordionItem).toBeDefined();
		expect(accordionItem).toMatchSnapshot();
	});
});
