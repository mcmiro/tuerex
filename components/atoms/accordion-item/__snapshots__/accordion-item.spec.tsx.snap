// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test AccordionItem component Should pass when AccordionItem is defined 1`] = `[]`;
